admin/docker-compose-nginx.yml

```diff
@@ -0,0 +1,56 @@
+version: '3.4'
+services:
+  onlineconf-admin:
+    build: .
+    image: onlineconf-admin
+    pull_policy: build
+    depends_on:
+      onlineconf-database:
+        condition: service_healthy
+    environment:
+      ONLINECONF_DATABASE_HOST: onlineconf-database
+      ONLINECONF_DATABASE_BASE: onlineconf
+      ONLINECONF_DATABASE_USER: onlineconf
+      ONLINECONF_DATABASE_PASSWORD: onlineconf
+      ONLINECONF_AUTH_HEADER: X-Forwarded-User
+    restart: always
+    volumes:
+      - staticshare:/var/www/onlineconf-admin
+  onlineconf-database:
+    image: mysql:8.0
+    volumes:
+      - data:/var/lib/mysql
+      - ./etc/charset.cnf:/etc/mysql/conf.d/charset.cnf
+      - ./etc/schema.sql:/docker-entrypoint-initdb.d/1-schema.sql
+      - ./etc/notification.sql:/docker-entrypoint-initdb.d/2-notification.sql
+#      - ./etc/auth.sql:/docker-entrypoint-initdb.d/3-auth.sql
+#      - ./etc/example-auth.sql:/docker-entrypoint-initdb.d/4-example-auth.sql
+      - ./etc/empty.sql:/docker-entrypoint-initdb.d/5-empty.sql
```

Comment:
I think the only difference between standalone and nginx versions of docker-compose must be using of nginx (and authorization, optionally).

Reply:
Yes, it's my mistake